rexml.gemspec

```diff
@@ -57,5 +57,6 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake"
+  spec.add_development_dependency "benchmark_driver"
   spec.add_development_dependency "test-unit"
```

Comment:
Could you keep this list in alphabetical order?

Reply:
OK. I'll fix it!
Thanks!